repo_resource/requirements.txt

```diff
@@ -1,2 +1,3 @@
-gitrepo==2.31.1
+gitrepo==2.32.2
```

Comment:
why are these updates needed? I think this could be done in a seperate patch

Reply:
Keeping dependencies up to date
Created a separate commit